packages/aragon-wrapper/src/index.js

```diff
@@ -68,9 +68,18 @@ export default class Aragon {
     }
     options = Object.assign(defaultOptions, options)
 
+    // Check for required options
+    if(options.apm === undefined){
+      options.apm = []
+    }
+
+    if(options.ensRegistryAddress === undefined){
+      options.ensRegistryAddress = []
```

Comment:
I think both of these could be defined in the `defaultOptions` object?

As well, I think `options.apm` should default to an empty object, and `options.ensRegistryAddress` should default to a string.